### ISSUE TITLE
[Feature Request] Clear Feed from selected player

### ISSUE THREAD

**Issue by MinerSpark** [pypy-vrc/VRCX#266]
I often get players that run 2 games at once, Causing world switch spam in the feed from them.
Maybe we could get a feature that allows you to delete logs from the feed of Only that person, Or from whoever you've entered into the search bar?

**Comment by GenesisAria** [comment]
Or have it not continuously re-update unless it detects a new location from prior (not double-post).

**Comment by Natsumi-sama** [comment]
This wont work becuase if you switch world then switch back it would show the wrong world, there is no simple way of doing this that won't cause bugs. I was planning on making a user blacklist for the feed but that's not happening any time soon, have to make local user favorites first.

**Comment by GenesisAria** [comment]
Yeah i thought again after you said that and realized you'd need to program an AI for that.

Perhaps another possibility is a flood detection, if it's changing too quickly for one person, slow down update intervals for them?

**Comment by Natsumi-sama** [comment]
It's mostly event based from the web socket and ignoring incoming events will likely cause bugs.

**Comment by Chi-Yu** [comment]
I just ran into this issue again now that idle game worlds are so popular. People run the game twice to farm idle game points.

My feed is basically unusable now.
